src/components/ui/form/Button/Button.jsx

```diff
@@ -0,0 +1,30 @@
+import classNames from "classnames";
+import PropTypes from "prop-types";
+import "./Button.scss";
+
+function Button({ children, className, type, ...rest }) {
+    return (
+        <button
+            type={type}
+            className={classNames({
+                btn: true,
```

Comment:
if `btn` is always true, you don't have to use it in any condition. You can write it like
className={classNames('btn'{.....}